src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -86,6 +118,15 @@ public void caseDataDefaults(CaseData caseData) {
         struckOutDefaults(caseData);
         dateToCurrentPosition(caseData);
         flagsImageFileNameDefaults(caseData);
+        setGlobalSearchDefaults(caseData);
+    }
+
+    public void setGlobalSearchDefaults(CaseData caseData) {
+        if (featureToggleService.isGlobalSearchEnabled()) {
+            setCaseNameHmctsInternal(caseData);
+            setCaseManagementLocation(caseData);
+            setCaseManagementCategory(caseData);
+        }
```

Comment:
Would prefer to use guard clauses for feature toggling as a pattern going forward. This method isnt that large but this pattern can help reduce cognitive complexity in the future as more devs adopt this.

```suggestion
        if (featureToggleService.isGlobalSearchEnabled()) {
            return;
        }
        setCaseNameHmctsInternal(caseData);
        setCaseManagementLocation(caseData);
        setCaseManagementCategory(caseData);
```